client/errors/reducer.js

```diff
@@ -0,0 +1,46 @@
+import {
+  RECEIVE_CLIENT_LOCATION_SUCCESS,
+  RECEIVE_CLIENT_LOCATION_ERROR,
+  RECEIVE_MARKER_LOCATION_SUCCESS,
+  RECEIVE_MARKER_LOCATION_ERROR,
+  DRAW_POLYGON_SUCCESS,
+  DRAW_POLYGON_ERROR
+} from "../components/Map/actions"
+
+import {
+  RECEIVE_BOUNDARIES_SUCCESS,
+  RECEIVE_BOUNDARIES_ERROR,
+} from "../boundaries/actions"
+
+export const initialState = {}
+
+export const reducer = (state = initialState, action) => {
+  Object.freeze(state)
+  const newState = { ...state }
+  switch (action.type) {
+    case RECEIVE_CLIENT_LOCATION_SUCCESS:
+      delete newState[RECEIVE_CLIENT_LOCATION_ERROR]
+      return newState;
+    case RECEIVE_MARKER_LOCATION_SUCCESS:
+      delete newState[RECEIVE_MARKER_LOCATION_ERROR]
+      return newState;
+    case RECEIVE_BOUNDARIES_SUCCESS:
```

Comment:
Does it make sense to handle the success results here too in the error cases? 

A little confused how `delete newState[DRAW_POLYGON_ERROR]` works in the success case. Wouldn't that just be handled in the success case in the polygon reducer?

Reply:
Yes. This reducer is handling the displaying and clearing of error messages. When errors occur, they will be added to this part of the state. We must then also have the opposite effect of removing said errors when successes occur.